app/controllers/posts_controller.rb

```diff
@@ -1,17 +1,12 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = Post.where(author_id: @user.id)
+    @posts = @user.posts
```

Comment:
- [ ] Great job working on the functionality and tests so far but I think it would be great if you fix the N+1 problem since I couldn't see that you were able to solve it when fetching all posts and their comments for a user and this functionality should be here in posts#index since here we are fetching all posts and their comments 🔥♻🚀

- I could see that you tried to solve it in posts#show but you commented on the code.